operator/roles/kiali-remove/vars/main.yml

```diff
@@ -5,3 +5,10 @@ kiali_vars:
     {%- else -%}
     {{ kiali_defaults.deployment }}
     {%- endif -%}
+  auth: |
```

Comment:
trivial - keep alphabetical ordering convention - move this "auth" block above the "deployment" block